libs/features/georoutingv2/georoutingv2.css

```diff
@@ -78,7 +78,7 @@
   background: initial;
 }
 
-.dialog-modal.locale-modal-v2 .tabs div[role="tablist"] .tabList-container {
+.dialog-modal.locale-modal-v2 .tabs div[role="tablist"] .tab-list-container {
```

Comment:
There are `georoutingv2` unit-test failures, not sure if this is related to these changes? Can you please check?

Reply:
Glad we have tests. I'm seeing an issue at https://rparrish-tabs-update--milo--adobecom.hlx.page/about/?akamaiLocale=CH
Looking into it...

Reply:
I figured out the issue and have pushed up a fix. The scope of the `rootElement` changes in a fragment and was throwing off a selector in the associated tab query. I added this use case in the desc.

Reply:
Thanks @ryanmparrish! However, the links in the description under `Test Tabs in Frags` doesn't use a document with fragments at all.

Reply:
Yes, geo routing launches a modal. Not a fragment embedded in the page per se. But, upon detection. 
https://github.com/adobecom/milo/blob/main/libs/features/georoutingv2/georoutingv2.js#L280